perl_lib/EPrints/Extras.pm

```diff
@@ -304,7 +304,7 @@ sub render_possible_doi
 
 	$value = "" unless defined $value;
 	if( $value =~ m!^
-			(?:http://(?:dx\.)?doi\.org/)?  # add this again later anyway
+			(?:http(?:s)://(?:dx\.)?doi\.org/)?  # add this again later anyway
```

Comment:
Possibly - I don't use this field, so couldn't test.  I was going on the approach that would make sure it didn't inadvertently become $1 and overwrite the value...